packages/synapse-react-client/package.json

```diff
@@ -43,10 +43,10 @@
     "@react-google-maps/api": "^2.18.1",
     "@react-hook/resize-observer": "^1.2.6",
     "@react-hookz/web": "^23.1.0",
-    "@rjsf/core": "^5.9.0",
-    "@rjsf/mui": "^5.9.0",
-    "@rjsf/utils": "^5.9.0",
-    "@rjsf/validator-ajv8": "^5.9.0",
+    "@rjsf/core": "5.12.1",
+    "@rjsf/mui": "5.12.1",
+    "@rjsf/utils": "5.12.1",
+    "@rjsf/validator-ajv8": "5.12.1",
```

Comment:
Bump while I'm here, and remove the `^` marker because we're relying on an experimental prop (for derived annotations)